Godot_test/ClientScripts/GameEventHandler.cs

```diff
@@ -0,0 +1,97 @@
+﻿using Dungeons.ASCIIDisplay;
+using Godot;
+using Roguelike.Abstract.Multimedia;
+using Roguelike.Abstract;
+using Roguelike;
+using Roguelike.Managers;
+using Roguelike.Multimedia;
+using System;
+using Roguelike.Events;
+using System.IO;
+using Dungeons.Tiles;
+using Roguelike.Tiles.LivingEntities;
+using static System.Net.WebRequestMethods;
+using NUnit.Framework;
+using System.Collections.Generic;
+using System.Linq;
+using Roguelike.Tiles.Interactive;
+using System.Drawing;
+using Dungeons.TileContainers;
+namespace God4_1.ClientScripts
+{
+  public class GameEventHandler
+  {
+    public void ActionsManager_ActionAppended(object sender, GameEvent ev)
+    {
+      if (ev is LivingEntityAction)
+      {
+        var lea = ev as LivingEntityAction;
+        if (lea.Kind == LivingEntityActionKind.Moved)
```

Comment:
Each if is evaluated, while only one is true, we shall use if/else if or switch of some  mechanism like type->function map